src/applications/appeals/996/components/InformalConference.jsx

```diff
@@ -102,16 +95,6 @@ export const InformalConference = ({
         updatePage(event);
       }
     },
-    onSelectionOriginal: event => {
-      const { value } = event?.detail || {};
-      if (value) {
-        const formData = {
-          ...data,
-          informalConference: value,
-        };
-        update(formData, value, informalConferenceLabels[value]);
-      }
-    },
     onSelectionNew: event => {
```

Comment:
Should this still be called `onSelectionNew`?

Reply:
Done 😛